test/integration/ga.ts

```diff
@@ -17,12 +17,98 @@
 
 import { describe, it, before } from 'mocha'
 import { expect } from 'chai'
-import { getSdk } from './'
+import { getSdk } from '.'
 import { generateKeyPair } from '../../src/utils/crypto'
 import { encode } from '../../src/utils/encoder'
 import MemoryAccount from '../../src/account/memory'
+// @ts-expect-error TODO Remove
 import { unpackTx } from '../../src/tx/builder'
 
+// TODO remove and import interfaces once contract aci is merged
+interface FunctionACI {
+  arguments: any[]
+  name: string
+  payable: boolean
+  returns: string
+  stateful: boolean
+}
+interface Aci {
+  encodedAci: {contract: {
+    name: string
+    event: any
+    kind: string
+    state: any
+    type_defs: any[]
+    functions: FunctionACI[]
+  }}
+  externalEncodedAci: any[]
+  interface: string
+}
+interface ContractInstance {
```

Comment:
You can reuse these types in the code while removing stamps in contracts section

Reply:
I mean, I see the same type of definitions in multiple places. Maybe we get to define them while removing stamps.  Not sure if we want to merge this now. we can merge this PR at last.

Reply:
Absolutely, these types are declared in the [PR ](https://github.com/aeternity/aepp-sdk-js/blob/7d19119e44fc9c2e135238d2775996ef2e1ee234/src/contract/aci.ts)regarding the RPC module. Once that's merged I will remove them from here and update the PR. Should I rename this to WIP in this case?

Reply:
I am resolving this. A simple to-do comment will solve this.